src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -2085,13 +2095,16 @@ FullWidthRepeat2:
 
             If Overflows Then
```

Comment:
FYI The order is now in sync with C#, an interesting case is an overflowing integer with a trailing underscore which gives 1 and 2 errors in VB and C# respectively. This is now fixed.